tests/vm/dune

```diff
@@ -0,0 +1,34 @@
+(library
+ (name vm_test)
+ (modules vm_test)
+ (libraries lambda_vm alcotest))
+
+(library
+ (name test_gas)
+ (inline_tests)
+ (modules test_gas)
+ (libraries vm_test lambda_vm)
+ (preprocess
+  (pps ppx_lambda_vm ppx_expect)))
+
+(test
+ (name test_simple_expr)
+ (modules test_simple_expr)
+ (libraries vm_test lambda_vm alcotest)
+ (preprocess
+  (pps ppx_lambda_vm)))
+
+(test
+ (name test_errors)
+ (modules test_errors)
+ (libraries vm_test lambda_vm alcotest)
+ (preprocess
+  (pps ppx_lambda_vm)))
+
+(test
+ (name test_prim)
+ (modules test_prim)
+ (libraries vm_test lambda_vm alcotest qcheck-alcotest)
+ (preprocess
+  (pps ppx_lambda_vm)))
```

Comment:
There is no need to put each module in it's own library, put everything on vm_test, it's good enough

Reply:
Oh ok, `test_gas` does need its own stanza though.

Reply:
Changed the way gas is tested base on your advice. Now it checks if gas is empty, also changed the way the check and burn gas works in order to have it empty at the and not needing an extra gas to work.